libraries/AP_HAL_ChibiOS/hwdef/JFB110/hwdef.dat

```diff
@@ -0,0 +1,360 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the JFB110 hardware
+# define AP_CUSTOM_FIRMWARE_STRING "(JFB110 V1.1.0)"
+
+# board ID for firmware load
+APJ_BOARD_ID 1110
+
+# MCU class and specific type
+MCU STM32H7xx STM32H755xx
+define CORE_CM7
+#define SMPS_PWR
+
+# crystal frequency 24MHz
+OSCILLATOR_HZ 24000000
+
+# the H755 has 128k sectors
+# bootloader is installed at 128kb offset
+FLASH_BOOTLOADER_LOAD_KB 128
+FLASH_RESERVE_START_KB 128
+FLASH_SIZE_KB 2048
+HAL_STORAGE_SIZE 32768
+
+# USB setup
+# USB_VENDOR 0x0A8E  # JAE
```

Comment:
if this is uncommented then also need a device ID and need a device driver for MissionPlanner and QGC on windows

Reply:
ok thanks.  we will leave this as the default USB_VENDOR then.